web/models.py

```
@@ -2195,47 +2210,85 @@ def __str__(self):
-                        score += question.points
-                elif question.question_type == "short":
-                    # Short answers require manual grading in this implementation
-                    # We could implement auto-grading logic here for simple cases
-                    pass
-            except QuizQuestion.DoesNotExist:
-                pass
-
-        self.score = score
-        self.max_score = max_score
-        self.save()
+        q_ids = [int(q) for q in answers.keys()]
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)